dbt_copilot_helper/commands/dns.py

```diff
@@ -439,14 +440,17 @@ def separate_hyphenated_application_environment_and_service(
     return domain_name, response["LoadBalancers"][0]
 
 
-@click.group()
+@click.group(cls=ClickDocOptGroup)
 def domain():
     pass
 
 
 @domain.command()
 @click.option(
-    "--domain-profile", help="aws account profile name for R53 domains account", required=True
+    "--domain-profile",
+    help="aws account profile name for R53 domains account",
+    required=True,
+    type=click.Choice(["dev", "live"]),
 )
 @click.option(
     "--project-profile", help="aws account profile name for certificates account", required=True
```

Comment:
```suggestion
    "--project-profile", help="AWS account profile name for certificates", required=True
```

Reply:
Resolved in https://github.com/uktrade/copilot-tools/pull/170/commits/10b7c4ed45e9d2a1b535d3c008f04228b2c5f310.